src/redux/Reservations/reservations.js

```diff
@@ -0,0 +1,71 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+// import axios from 'axios';
+
+// const apiURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/q4jqYH5waNwpAMlI8Ds9/books';
+const initialState = [
+  {
+    id: 1,
+    user: {
+      id: 1,
+      name: 'Martha Vivatoska',
+    },
+    accommodation: 'This will be the accommodation description',
+    start_date: '22/11/20',
+    end_date: '22/12/20',
+  },
+  {
+    id: 1,
+    user: {
+      id: 1,
+      name: 'Martha Vivatoska',
+    },
+    accommodation: 'This will be the accommodation description',
+    start_date: '22/11/20',
+    end_date: '22/12/20',
+  },
+];
+
+// export const reserve = createAsyncThunk('reservations/reserve', async (reservation) => {
+// const response = await axios.post(apiURL, reservation);
+// if (response.status === 201) {
+//   return reservation;
+// }
+// return response.data;
+// });
```

Comment:
- I invite you to consider don't stay commented lines on the project, this is a good practice. If you would like to have notes for code, you could always use the GitHub history.